### PR TITLE
Hacktoberfest - Advice Generator app

### PR DESCRIPTION
**Pull request by curator69**
Generate advice by clicking the button, built using fetch API.

### CODE REVIEW

**Review by Anamaya1729 — CHANGES_REQUESTED** [review]
Improve the readme.

**Review by Anamaya1729 — CHANGES_REQUESTED** [review]
Change the heading. Add your name under contributor heading.

**Comment by curator69** [comment]
Sorry this is my first contribution, can u tell me what I have to do exactly to the heading, also i have added my name and github profile link.

File: easy/advice_generator-app/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+This app generates an advice using fetch API when the button is clicked.
+
+### Tech Stack
+
+- HTML
+- CSS
+- JavaScript
+
+### Maintainer
```
Comment (Anamaya1729) [inline easy/advice_generator-app/README.md:17]: Change this to Contributor. Everything else is okay..

**Review by Anamaya1729 — COMMENTED** [review]
Just change line 17 of README.md from Maintainer to Contributor..

File: easy/advice_generator-app/README.md (hunk 1 above)
Comment (curator69) [reply easy/advice_generator-app/README.md]: done

**Review by Anamaya1729 — APPROVED** [review]
hacktoberfest-accepted

**Comment by Anamaya1729** [comment]
Thanks for your contribution. Please star the repository to support it.

**Comment by curator69** [comment]
thank you so much!

**Pull request merged**